modules/catalog-compaction/src/main/java/org/apache/ignite/internal/catalog/compaction/CatalogManagerCompactionFacade.java

```diff
@@ -38,6 +38,7 @@ class CatalogManagerCompactionFacade {
         this.catalogManager = catalogManager;
     }
 
+    // TODO https://issues.apache.org/jira/browse/IGNITE-22522 Remove this method.
```

Comment:
I was wrong, we will not remove this method on "merging" colocation to main as we've decided to stick with table based minTimeCollectorService.